src/content/service.js

```diff
@@ -90,7 +90,7 @@ module.exports = async ({ botfile, projectLocation, logger }) => {
     }
 
     const items = await query
-      .orderBy('createdOn')
```

Comment:
It was `camelCase` here but is `snake_case` above

Reply:
to clarify that camel_case default will not work as the column is actually camelCased here

Reply:
Yeah, it's what I missed while merging in latest changes. Fixed that.